microscopy_SPIM001/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We can remove the LICENSE file (we will go with CC0 license instead of MIT, no license file required).